config/source_blockchain.go

```diff
@@ -16,15 +16,15 @@ import (
 // Specifies the height from which to start processing historical blocks.
 type SourceBlockchain struct {
 	SubnetID                          string                           `mapstructure:"subnet-id" json:"subnet-id"`
-	BlockchainID                      string                           `mapstructure:"blockchain-id" json:"blockchain-id"`
+	BlockchainID                      string                           `mapstructure:"blockchain-id" json:"blockchain-id"` //nolint:lll
```

Comment:
Not for this PR, but we should look into if we still need both mpastructure and json tags. IIRC, the json tag is only used in tests.